src/sc/sidechain.h

```diff
@@ -155,4 +155,47 @@ namespace Sidechain {
     bool hasScCreationOutput(const CTransaction& tx, const uint256& scId);
 }; // end of namespace
 
+
+struct CMinimalSidechain //Sidechain with minimal info, to be included in wallet
```

Comment:
Consider renaming to CWalletSidechain

Reply:
I see the value of the suggestion. Let's discuss during final code review